CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Hardening: upgrade microhttpd dependency from 0.9.70 to 0.9.76
```

Comment:
Small typo:

```suggestion
- Hardening: upgrade microhttpd dependency from 0.9.73 to 0.9.76
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4321/commits/de7216338106e7d5c3068ab2f1d702e37fa4e89e